backend/core/models.py

```diff
@@ -57,9 +55,11 @@ def save(self, *args, **kwargs):
             self.url = f"{HOST}/authors/{self.id}"
         super().save(*args, **kwargs)
 
+
 class Relation(AbstractModel):
     to_author = models.ForeignKey(Author, related_name='follower', on_delete=models.CASCADE)
     from_author = models.ForeignKey(Author, related_name='following', on_delete=models.CASCADE)
+    approved = models.BooleanField(default=False)
```

Comment:
Why do we need approved here? Like if a request get's approved then the from_author_request or to_author_request would be set to 1

Reply:
I just added it to not forget about the approval feature. We can discuss what to do when we work on it.